diff --git a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_core.d.ts b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_core.d.ts
index da4695c..7fec26b 100644
--- a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_core.d.ts
+++ b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_core.d.ts
@@ -555,6 +555,7 @@ declare class Graphics {
     static boxHeight: number;
     static scale: number;
 
+    static printFullError(name: string, message: string, stack: string): void;
     static initialize(width: number, height: number, type: string): void;
     static tickStart(): void;
     static tickEnd(): void;
@@ -593,7 +594,7 @@ declare class Graphics {
     static _scale: number;
     static _realScale: number;
     static _errorShowed: boolean;
-    static _errorPrinter: HTMLParagraphElement;
+    static _errorPrinter: HTMLParagraphElement | null;
     static _canvas: HTMLCanvasElement;
     static _video: HTMLVideoElement;
     static _videoUnlocked: boolean;
diff --git a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_managers.d.ts b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_managers.d.ts
index 8ee1fbb..b63d6bb 100644
--- a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_managers.d.ts
+++ b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_managers.d.ts
@@ -1007,6 +1007,7 @@ declare class BattleManager {
     static _escaped: boolean;
     static _rewards: IDataRewards;
 
+    static refreshEnemyReinforcements(): void;
     static setup(troopId: number, canEscape: boolean, canLose: boolean): void;
     static initMembers(): void;
     static isBattleTest(): boolean;
diff --git a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_objects.d.ts b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_objects.d.ts
index 5ff8274..03fe65c 100644
--- a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_objects.d.ts
+++ b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_objects.d.ts
@@ -16,6 +16,9 @@ declare class Game_Temp {
     _commonEventId: number;
     _destinationX: number;
     _destinationY: number;
+    _previousTeleportX: number;
+    _previousTeleportY: number;
+    _previousTeleportMap: number;
 }
 
 declare class Game_System {
@@ -175,7 +178,7 @@ declare class Game_Variables {
     initialize(): void;
     clear(): void;
     value(variableId: number): number;
-    setValue(variableId: number, value: number): number;
+    setValue(variableId: number, value: number|string): number;
     onChange(): void;
 
     _data: number[];
@@ -976,7 +979,9 @@ declare class Game_Enemy extends Game_Battler {
     _screenX: number;
     _screenY: number;
 
-    constructor();
+    constructor(id: number, x: number, y: number);
+    setTroopId(id: number): void;
+    setTroopMemberId(id: number): void;
     initialize(): void;
     initialize(enemyId: number, x: number, y: number): void;
     initMembers(): void;
@@ -1155,6 +1160,7 @@ declare class Game_Troop extends Game_Unit {
     _interpreter: Game_Interpreter;
     _turnCount: number;
     _enemies: Game_Enemy[];
+    _newEnemies: Game_Enemy[];
     _troopId: number;
     _eventFlags: { [key: number]: boolean };
     _namesCount: { [key: string]: number };
@@ -1178,6 +1184,8 @@ declare class Game_Troop extends Game_Unit {
     goldTotal(): number;
     goldRate(): number;
     makeDropItems(): IDataItem[];
+    addTroopReinforcements(id: number): void;
+    addTroopReinforcementsWithRelativePosition(id: number, x: number, y: number): void;
 }
 
 declare class Game_Map {
diff --git a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_scenes.d.ts b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_scenes.d.ts
index f79ec5a..0cda5fe 100644
--- a/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_scenes.d.ts
+++ b/node_modules/rpgmakermv_typescript_dts/typings/rpgmakermv/rpg_scenes.d.ts
@@ -5,6 +5,9 @@ declare class Scene_Base extends Stage {
     _fadeSprite: ScreenSprite;
     _windowLayer: WindowLayer;
     _spriteset: Spriteset_Base;
+    _mapLoaded: boolean;
+    _messageWindow: Window_Message;
+    _isTeleporting: boolean;
 
     constructor();
     initialize(): void;
